merlin/systems/dag/ops/workflow.py

```diff
@@ -69,68 +61,19 @@ def __init__(
         super().__init__()
 
         self.workflow = workflow
+        if label_columns:
+            self.workflow = workflow.remove_inputs(label_columns)
         self._nvt_model_name = None
         self.sparse_max = sparse_max or {}
         self.max_batch_size = max_batch_size
         self.label_columns = label_columns or []
         self.model_framework = model_framework or ""
         self.cats = cats or []
         self.conts = conts or []
-        self._python = backend == "python"
 
-        if workflow is not None:
-            self.input_schema = workflow.input_schema
-            self.output_schema = workflow.output_schema
-
-    def transform(self, col_selector: ColumnSelector, transformable: Transformable):
```

Comment:
removed some of the basic methods because they require refactor for the local runtime. And this PR focuses on the triton runtime migration. Will readd this functionality when we start working on local runtimes. Meanwhile the basic classes will rely on base class function definitions which raise not implemented errors.